app/controllers/admin/edition_legacy_associations_controller.rb

```diff
@@ -2,11 +2,10 @@ class Admin::EditionLegacyAssociationsController < Admin::BaseController
   before_action :find_edition
   before_action :enforce_permissions!
   before_action :limit_edition_access!
-  layout :get_layout
+  layout "design_system"
 
   def edit
     @path = get_path
-    render_design_system("edit", "edit_legacy", next_release: true)
```

Comment:
Can we also delete the view file `views/admin/edition_legacy_associations/edit_legacy.html.erb` now?

Reply:
Sorry thought i'd deleted it!